APMrover2/Rover.h

```diff
@@ -532,6 +512,9 @@ class Rover : public AP_HAL::HAL::Callbacks {
 
     // Simple mode
     float simple_sin_yaw;
+
+    // sailboat enabled
+    bool get_sailboat_enable() { return g2.sailboat.enabled(); }
```

Comment:
const  function ?

Reply:
fixed, thanks

Reply:
I had to un-const this, it was giving a compiler warning